modules/HomePage/pages/home-page.tsx

```diff
@@ -13,6 +14,17 @@ interface WidgetData {
 }
 
 const HomePage: FC<IPostFeed.IPosts> = ({ data }): ReactElement => {
+  const router = useRouter();
+  useEffect(() => {
+    if (document.cookie.split('=')[0] !== 'user') {
+      router
+        .push('/login')
+        .then()
```

Comment:
we discussed this earlier. this is immature way of handling promises.

Reply:
I tried to `return router.push('/login')` but there was an error regarding useEffect, I will investigate it further.